core/eolearn/tests/test_core_tasks.py

```diff
@@ -148,28 +149,68 @@ def test_io_task_pickling(filesystem, task_class):
     assert isinstance(unpickled_task, task_class)
 
 
-def test_add_rename_remove_feature(patch):
+def test_add_feature(patch: EOPatch) -> None:
     cloud_mask = np.arange(10).reshape(5, 2, 1, 1)
-    feature_name = "CLOUD MASK"
-    new_feature_name = "CLM"
+    feature = (FeatureType.MASK, "CLOUD MASK")
 
-    patch = copy.deepcopy(patch)
+    with pytest.raises(KeyError):
+        patch[feature]
 
-    patch = AddFeatureTask((FeatureType.MASK, feature_name))(patch, cloud_mask)
-    assert np.array_equal(patch.mask[feature_name], cloud_mask), "Feature was not added"
+    patch = AddFeatureTask(feature)(patch, cloud_mask)
+    assert np.array_equal(patch[feature], cloud_mask)
 
-    patch = RenameFeatureTask((FeatureType.MASK, feature_name, new_feature_name))(patch)
-    assert np.array_equal(patch.mask[new_feature_name], cloud_mask), "Feature was not renamed"
-    assert feature_name not in patch[FeatureType.MASK], "Old feature still exists"
 
-    patch = RemoveFeatureTask((FeatureType.MASK, new_feature_name))(patch)
-    assert feature_name not in patch.mask, "Feature was not removed"
+@pytest.mark.parametrize(
+    "feature, feature_data",
+    [
+        ((FeatureType.BBOX, None), BBox((24.54, 56.45, 95.4, 13.43), CRS(3857))),
+        ((FeatureType.TIMESTAMP, None), [datetime(2022, 1, 1, 10, 4, 7), datetime(2022, 1, 4, 10, 14, 5)]),
+    ],
+)
+def test_add_bbox_timestamps(feature: FeatureSpec, feature_data: np.ndarray) -> None:
```

Comment:
I don't see any reason why the `add` functionality should be split for bbox/timestamps and for other features. Couldn't you add a new parametrization entry for the cloud mask from L153 here? The logic is the same:
- create empty patch
- check that the feature doesn't exist
- add new feature with feature data
- check if feature of eopatch is the same as feature data
The same assertion (`assert patch[feature] == feature_data`) should work, no need to use `np` specific checks.

Reply:
same for testing the remove functionality

Reply:
updated